iOS Example/iOS Example/AppDelegate.swift

```diff
@@ -7,4 +7,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Override point for customization after application launch.
         true
     }
+
+    // Handle redirection after OAuth completed or failed
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        guard let vc = (window?.rootViewController as? ViewController),
+              let authorizeRedirectionHandler = vc.authorizeRedirectionHandler else {
+            return true
+        }
+        print("[OAuth]: Redirect back to app")
+        authorizeRedirectionHandler(url)
+        return true
+    }
```

Comment:
A typical API design is to add this method to Authgear. So the user of the SDK just forwards the call.

Reply:
Setting `authorizeRedirectionHandler` in Authgear, then call this by `container.authorizeRedirectionHandler` ?

Reply:
I mean we add `func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool` to Authgear. Thus the developer just need to write

```swift
func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
  // Assume the developer has defined authgear as a property of AppDelegate.
  self.authgear.application(app, open: url, options: options)
}
```

Reply:
I see, will add this as well